ui/guest/src/utils/contentType.ts

```diff
@@ -75,9 +76,20 @@ export function getDefaultValue(field: ContentTypeField): string {
         return 'https://via.placeholder.com/150';
```

Comment:
Try using image dimensions from content type definition to set the right pic size